### PR TITLE
feat(backend): adds a name field for the peer model

### LINKED ISSUE #827: Add a name field for the Peer model in the GraphQL schema
Currently the GraphQL schema model for a Peer is:
```
type Peer implements Model {
  id: ID!
  maxPacketAmount: UInt64
  http: Http!
  asset: Asset!
  staticIlpAddress: String!
  createdAt: String!
}
```
An optional `name` field needs to be added so that there is a field which is easy to use and remember when looking at, or searching for specific peers.

### PR DESCRIPTION
**Pull request by dragosp1011**
## Changes proposed in this pull request

- adds a name field for the peer model

## Context

fixes #827 

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

File: packages/backend/src/peer/model.ts [hunk 1]
```diff
@@ -46,6 +46,8 @@ export class Peer
 
   public staticIlpAddress!: string
 
+  public name?: string
```
Comment (wilsonianb) [inline packages/backend/src/peer/model.ts:49]: The column needs to be added to `peers` table.
You can add it to the migration file:
https://github.com/interledger/rafiki/blob/4db6f8e8ee6e4812af452af379855e4c3ff8d2f6/packages/backend/migrations/20211026133430_create_peers_table.js#L2
The create peer insert should also include `name`
https://github.com/interledger/rafiki/blob/4db6f8e8ee6e4812af452af379855e4c3ff8d2f6/packages/backend/src/peer/service.ts#L103-L109

**Review by wilsonianb — COMMENTED** [review]
Add `name` to peer service and resolver tests
https://github.com/interledger/rafiki/blob/main/packages/backend/src/peer/service.test.ts
https://github.com/interledger/rafiki/blob/main/packages/backend/src/graphql/resolvers/peer.test.ts

**Review by mkurapov — APPROVED** [review]
looks good!

File: packages/backend/src/peer/service.test.ts [hunk 2]
```diff
@@ -66,7 +67,8 @@ describe('Peer Service', (): void => {
             endpoint: faker.internet.url()
           }
         },
-        staticIlpAddress: 'test.' + uuid()
+        staticIlpAddress: 'test.' + uuid(),
+        name: faker.name.fullName()
```
Comment (wilsonianb) [inline packages/backend/src/peer/service.test.ts:71]: You could remove `name` here and just test it in the `all settings` test below.

**Review by wilsonianb — APPROVED** [review]

**Pull request merged**